src/views/posts-index.js

```diff
@@ -0,0 +1,44 @@
+var RunList = Backbone.View.extend({
+  model: null,
+
+  tagName: 'li',
+
+  initialize() {
+    // Wire up model events
+  },
+
+  render() {
+    // Sets up the DOM
+    this.$el.html(`
+      ${this.model.get('date')} - ${this.model.get('time')}
+
+        <a href="#"><i class="fa fa-plus-circle"></i></i></a>
```

Comment:
Your link to the `new` route should have `href="#new"` and this should be on the list. So you could append an `li` with this link after your foreach in the actual list views.